man/adoc/bpftrace.adoc

```diff
@@ -1318,13 +1324,15 @@ hardware:cache-misses:1e6 { @[pid] = count(); }
 * `interval:us:count`
 * `interval:ms:count`
 * `interval:s:count`
+* `interval:count`
```

Comment:
Which option should users prefer? Here and below. It's unclear if there's a recommendation.